src/docs/about.md

```diff
@@ -0,0 +1,186 @@
+---
+title: What is Broccoli?
+---
+
+Broccoli is a simple JavaScript build tool, that allows you to configure your build pipeline in JavaScript (like
+you're already used to writing), and handles setting up the filesystem state for each transformation that's 
```

Comment:
I would not necessarily assume that readers understand the distinction between an asset pipeline and a module bundler.

You could say something like

> Broccoli is a build tool, that through allowing developers to compose modular plugins to form an asset pipeline, quickly and reliably combines and transforms source files into deployable static assets.